### DISCUSSION TITLE
Unable to render no rows view

### DISCUSSION THREAD

**Discussion by rohit-squirrel** [Comcast/react-data-grid#3080] [Q&A]
Hi, i am unable to view no rows view in my data grid. I am using filters and sort also. I followed the example provided in the website for no rows as they have mentioned then also i am not able to show it. It not even going into that function. Any help is appreciated.
Thanks!

react version : 18.0
react-data-grid version : 7.0.0-beta.12

**Comment by vishal-vc** [comment]
renderers={{ noRowsFallback: <EmptyRowsRenderer /> }}

above prop is not working @amanmahajan7

**Comment by lucasalmeida92** [comment]
Same here, not working :/
